.travis.yml

```diff
@@ -42,7 +42,7 @@ before_install:
     - npm prune
     - pip install -U pip virtualenv
 install:
-    - pip install -r requirements.txt -r requirements-dev.txt -e .
+    - pip install -r clients/python/requirements.txt -r requirements-dev.txt -e .
```

Comment:
Maybe this should be:

    - pip install -r requirements-dev.txt -e . -e clients/python

So we can test out our setup.py